### ISSUE TITLE
Can't create Security Group rule since v1.10.0

### ISSUE THREAD

**Issue by mcanevet** [terraform-providers/terraform-provider-openstack#450]
This [this](https://github.com/terraform-providers/terraform-provider-openstack/commit/e2468a4c7544e0b8821c1334fc21f993c7a89513#diff-40a9d194741f92d9c6532e4ccea6c1af) has been merged I get this error message on my cloud provider:

```
Bad request with: [POST https://network.compute.sbg3.cloud.ovh.net/v2.0/security-group-rules], error message: {"NeutronError": {"message": "Unrecognized attribute(s) 'description'", "type": "HTTPBadRequest", "detail": ""}}
```

**Comment by mcanevet** [comment]
@yanndegat this is on OVH's SBG3 region any idea?

**Comment by jtopjian** [comment]
Sorry about this. A patch for this has been merged into master and will be available in 1.11.0.

**Comment by mcanevet** [comment]
@jtopjian OK thanks.
